core/src/test/java/net/sourceforge/jnlp/TinyHttpdImpl.java

```diff
@@ -136,8 +135,8 @@ public void run() {
                     String request = t.nextToken();
                     String filePath = t.nextToken();
 
-                    boolean isHeadRequest = request.equals(ResourceTracker.RequestMethods.HEAD.toString());
-                    boolean isGetRequest = request.equals(ResourceTracker.RequestMethods.GET.toString());
+                    boolean isHeadRequest = request.equals(HttpMethod.HEAD.name());
```

Comment:
we should avoid to use `a.equals(b)` and use `Objects.equals(a, b)` instead. By doing so no `NPE` will happen if a is null.

Reply:
Good point, fixed.